README.md

```diff
@@ -406,6 +406,22 @@ jobs:
       - uses: psf/black@stable
 ```
 
+## Spotless
+
+If you happen to be using Gradle, the
+[Spotless](https://github.com/diffplug/spotless/tree/main/plugin-gradle) plugin has
+support for black. Spotless gives you the option to
+
+- [ratchet](https://github.com/diffplug/spotless/tree/main/plugin-gradle#ratchet) your
+  formatting so that only files which have changed since `origin/main` get formatted
+- slurp copyright info from
+  [git history](https://github.com/diffplug/spotless/tree/main/plugin-gradle#license-header)
+- seamlessly integrate with
```

Comment:
Integrations with formatters for other languages don't feel relevant to Black's documentation.

Reply:
As I see it, someone who comes to your website is saying "I want to stop caring about formatting", not "I want to stop caring about formatting just my python, but keep caring about my html templates, c code,  and markdown documentation".  So I think it's relevant that if you use Spotless as your integration layer, you only have one interface to worry about, versus "I'll use project.toml to format my python, precommit for my docs, and a shell-script to clang-format for my c-code."

The fact that this project relies on prettier for formatting its docs is a good example that the people who bother to use one autoformatter usually want others, and it's nice to be able to configure them all from one place, which is the whole point of Spotless.

One argument for keeping Spotless as a peer to VCS is that it offers unique features related to slurping info from git history, which can be valuable for projects with strict legal requirements, especially polyglot projects.  Regardless, I'll move Spotless to editor integration, and you can pick whichever commit you prefer :)